mslib/msui/mscolab.py

```diff
@@ -985,29 +990,41 @@ def browse():
                 self.add_proj_dialog.selectedFile.setText(file_name)
 
         self.proj_diag = QDialog()
-        self.add_proj_dialog = add_operation_ui.Ui_addOperationDialog()
+        self.add_proj_dialog = msc_add_operation_ui.Ui_addOperationDialog()
         self.add_proj_dialog.setupUi(self.proj_diag)
         self.add_proj_dialog.f_content = None
-        self.add_proj_dialog.buttonBox.accepted.connect(self.add_operation)
+        self.add_proj_dialog.buttonBox.accepted.connect(self.add_operation_from_new_dialogue)
         self.add_proj_dialog.buttonBox.button(QtWidgets.QDialogButtonBox.Ok).setEnabled(False)
         self.add_proj_dialog.path.textChanged.connect(check_and_enable_operation_accept)
         self.add_proj_dialog.description.textChanged.connect(check_and_enable_operation_accept)
         self.add_proj_dialog.category.textChanged.connect(check_and_enable_operation_accept)
         self.add_proj_dialog.browse.clicked.connect(browse)
         self.add_proj_dialog.category.setText(config_loader(dataset="MSCOLAB_category"))
+        self.add_proj_dialog.optFileBox.setHidden(True)
+        if name is not None:
+            self.add_proj_dialog.path.setText(name)
+        if description is not None:
+            self.add_proj_dialog.description.setText(name)
+        if xml is not None:
+            self.add_proj_dialog.f_content = xml
 
         # sets types from defined import menu
         import_menu = self.ui.menuImportFlightTrack
         for im_action in import_menu.actions():
             self.add_proj_dialog.cb_ImportType.addItem(im_action.text())
         self.proj_diag.show()
 
+    def add_operation_from_new_dialogue(self):
+        logging.debug("add_operation_from_dialogue")
```

Comment:
the name is difficult later to understand or to map to the origin. maybe use same writing (add_operation_from_new_dialog)

self.add_proj_dialog.buttonBox.accepted.connect(self.add_operation_from_new_dialogue)